src/Admin/MaintainAuthUsers/containers/AuthUserSearchValidation.js

```diff
@@ -3,6 +3,14 @@ const validateSearch = (user, handleError) => {
     handleError({ response: { data: [{ targetName: 'user', text: 'Enter a username or email address' }] } })
     return false
   }
+  if (!user.includes('@') && !/^[a-zA-Z0-9_]*$/.test(user.trim())) {
```

Comment:
since we are using a path variable for username, spring will reject any requests that contain certain special characters.  Therefore better to just validate here